<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToolBarCommandWidget</class>
 <widget class="QWidget" name="ToolBarCommandWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>305</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>240</width>
    <height>50</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="redoButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../앞으로가기.png</normaloff>../앞으로가기.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="undoButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../뒤로가기.png</normaloff>../뒤로가기.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="alleraseButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>모두 지우기</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../모두지우기.png</normaloff>../모두지우기.png</iconset>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
